### PR TITLE
[T2A2][T09-A3]NgHuanRan

### PR DESCRIPTION
**Pull request by shusiner**

### CODE REVIEW

**Comment by shusiner** [comment]
Ready for review

**Comment by shusiner** [comment]
Ready for review

**Comment by mauris** [comment]
@shusiner do you have a team number yet?

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -202,13 +202,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        renderInputTillExit();
```
Comment (z11i) [inline src/seedu/addressbook/AddressBook.java:205]: Yes here we can extract the method to keep the abstraction level single. Think about the other parts of your changes: do those affect the level of abstraction?

**Pull request closed without merging**